doc/architecture/decisions/0011-use-redis-backed-drafts.md

```diff
@@ -52,7 +52,7 @@ It would also be good if the solution could:
 
 ## Decision
 
-We’ll use the UI application’s existing Redis server as our storage. It allows us to store essentially unlimited amounts of data. The data is private to the current browsing session and is not accessible by other users.
+We’ll use the UI application’s existing Redis server as our storage. It allows us to store essentially unlimited amounts of data.
```

Comment:
what is the impact of this practically speaking? does it expose to exploits where other users draft data can be read?

Reply:
No practical implication – the access control is handled by the application instead (see "make sure that a user is not able to access data entered by a different user" further down).